lib/manager/maven/update.ts

```diff
@@ -44,3 +51,61 @@ export function updateDependency({
   }
   return `${spaces}${updatedContent}`;
 }
+
+export function bumpPackageVersion(
+  content: string,
+  currentValue: string | undefined,
+  bumpVersion: ReleaseType | string
+): BumpPackageVersionResult {
+  logger.debug(
+    { bumpVersion, currentValue },
+    'Checking if we should bump pom.xml version'
+  );
+  let bumpedContent = content;
+
+  if (!currentValue) {
+    logger.warn('Unable to bump pom.xml version, pom.xml has no version');
+    return { bumpedContent };
+  }
+
+  if (!semver.valid(currentValue)) {
```

Comment:
Are you sure this must be semver and not maven versioning?

Reply:
@viceice, are you sure it's maven versioning and not the versioning configured for this package?

Reply:
@viceice @astellingwerf This is the package version being bumped. Followed along the lines of other packet managers here that the bumped version must be a semver.